packages/firestore/src/local/shared_client_state.ts

```diff
@@ -408,37 +404,42 @@ export class WebStorageSharedClientState implements SharedClientState {
     return typeof window !== 'undefined' && window.localStorage != null;
   }
 
-  subscribe(sharedClientDelegate: SharedClientDelegate) {
-    this.sharedClientDelegate = sharedClientDelegate;
+  subscribe(syncEngine: SharedClientStateSyncer) {
+    this.syncEngine = syncEngine;
   }
 
+  // TODO(multitab): Handle user changes.
   start(initialUser: User, knownClients: ClientKey[]): void {
     assert(!this.started, 'WebStorageSharedClientState already started');
     assert(
-      this.sharedClientDelegate !== null,
+      this.syncEngine !== null,
       'Start() called before subscribing to events'
```

Comment:
"subscribing to events" is no longer accurate (especially with the getActiveClients change)